turbo/generators/config.ts

```diff
@@ -71,6 +102,12 @@ export default function generator(plop: PlopTypes.NodePlopAPI): void {
         name: 'description',
         message: 'Application description:',
       },
+      {
+        type: 'confirm',
+        name: 'runnableOnTeams',
+        message: 'Runnable to Teams?',
+        default: true,
```

Comment:
Could you explain what this does? (For my learning purposes)

Reply:
Yup, it just adds the `appPackage` folder with manifest in there.